src/pepr/operator/crd/sources/package/v1alpha1.ts

```diff
@@ -299,8 +299,32 @@ const sso = {
       attributes: {
         description: "Specifies attributes for the client.",
         type: "object",
-        additionalProperties: {
-          type: "string",
+        properties: {
+          "oidc.ciba.grant.enabled": {
+            type: "string",
+          },
+          "backchannel.logout.session.required": {
+            type: "string",
+          },
+          "backchannel.logout.revoke.offline.tokens": {
+            type: "string",
+          },
+          "post.logout.redirect.uris": {
+            type: "string",
+          },
+          "oauth2.device.authorization.grant.enabled": {
+            type: "string",
+          },
+          "pkce.code.challenge.method": {
+            type: "string",
+            enum: ["S256", "plain"],
+          },
+          "uds.core.groups": {
```

Comment:
I don't think this should ever be specified in the Package as a client attribute.  You might have done this because we use the type, but we probably don't want it to seem like this is a recommended way of setting the groups on a client.